refinery/file_store/models.py

```diff
@@ -34,9 +34,9 @@ def _mkdir(path):
     # https://stackoverflow.com/a/14364249
     try:
         os.makedirs(path)
-    except OSError:
+    except OSError as exc:
         if not os.path.isdir(path):
-            raise
+            raise RuntimeError(str(exc))
```

Comment:
Does this need to be wrapped as a string?

Reply:
Yes, we need to pass a string to RuntimeError. Explicit is better than implicit. https://www.python.org/dev/peps/pep-0020/